.travis.yml

```diff
@@ -74,6 +74,10 @@ before_install:
   - if [ ! -z "${_DL_UPSTREAM_GIT:-}" ]; then
       sudo tools/ci/install-latest-git.sh || { [ $? -eq 100 ] && exit 0; } || exit 1;
     fi
+  - sudo eatmydata apt-get install -y locales
+  - sudo sed -e 's,^# \(ru_RU.UTF-8\|bg_BG.UTF-8\),\1,g' /etc/locale.gen 
```

Comment:
One run on the most recent push passed (and the other is still running).  Missing an `-i`?

Reply:
Damn it, .. thank you!

Reply:
```suggestion
  - sudo sed -i -e 's,^# \(ru_RU.UTF-8\|bg_BG.UTF-8\),\1,g' /etc/locale.gen 
```